### DISCUSSION TITLE
Fixed height of table row not working in BIRT Viewer, but is working as a PDF export

### DISCUSSION THREAD

**Discussion by tnation1424** [eclipse-birt/birt#1573] [Q&A]
The problem with the Fix height of a column only happens when the Report is view in the BIRT Viewer, but for some reason when exporting as a PDF it works.  
BIRT Viewer shows lines 10 and 11.
![image](https://github.com/eclipse-birt/birt/assets/149100941/96e399bd-d8d3-4d6c-bb31-c4828d95f271)

This is the exporting of the PDF where it only shows up to line 9.  This is how we want it look in the BIRT Viewer.  Currently we are using Neon v4.6, which I know is a older version of BIRT.  I'm hoping that this is not a defect for this version of BIRT.
![image](https://github.com/eclipse-birt/birt/assets/149100941/4df9e02c-e150-4075-80e2-9c68db574148)

![image](https://github.com/eclipse-birt/birt/assets/149100941/6e06dc5d-4a24-4e58-8605-3fa6008824bd)

**Comment by speckyspooky** [comment]
Yes, the version 4.6 is an older version. The latest version is 4.14 (and upcoming 4.15), so many changes are done in the mean time.
It would help if you could provide a small example without data sources so that we are able to check it with newer versions.
Without a test case from your side it would be very hard to get the same constellation.

**Comment by tnation1424** [comment]
I apologize, I’m not too sure what kind of example you need? The .rptdesign?



Yes, the version 4.6 is an older version. The latest version is 4.14 (and upcoming 4.15), so many changes are done in the mean time.
It would help if you could provide a small example without data sources so that we are able to check it with newer versions.
Without a test case from your side it would be very hard to get the same constellation.

**Reply by speckyspooky** [reply]
Yes, the rptdesign would be helpfull with a table and static content from your side where you have the problem
and I would be able to test it from my side.

**Reply by tnation1424** [reply]
By Static content would a PDF work?  Sorry for all these questions.

**Reply by speckyspooky** [reply]
No, it would be necessary to have a version of your rptdesign but the text on it can be static (= fixed text).

**Comment by tnation1424** [comment]
Here is the ,rptdesign file. Thank you so much for your help.
[v2BO]triCommon__cstDDForm1354.rptdesign

**Reply by speckyspooky** [reply]
Sorry, please upload the rptdesign as a zipfile. Your file wasn't uploaded correctly.

**Comment by tnation1424** [comment]
[1354_m5.v1.15.4.zip](https://github.com/eclipse-birt/birt/files/14424370/1354_m5.v1.15.4.zip)

**Comment by hvbtup** [comment]
Your rptdesign file is not self-contained.
Thomas needs a report that _everybody_ can test.
The report must not reference Java classes or DB.
You can use the builtin example database as a data source, for example.
